test/buffer_manager_tests.cc

```diff
@@ -1098,6 +1098,14 @@ namespace detail {
 		}
 	}
 
+	TEST_CASE_METHOD(test_utils::runtime_fixture, "set buffer debug name", "[buffer_debug_name]") {
```

Comment:
```suggestion
	TEST_CASE_METHOD(test_utils::runtime_fixture, "buffer_manager allows to set debug names on buffers", "[buffer_manager]") {
```

I'm wondering whether this test should be using the buffer-manager APIs directly instead, given that its in "buffer_manager_tests" (right now it's more of an integration test).